packages/packages/libs/editor-controls/src/components/unstable-repeater/actions/tooltip-add-item-action.tsx

```diff
@@ -31,7 +33,8 @@ export const TooltipAddItemAction = ( {
 					size={ SIZE }
 					disabled={ disabled }
 					onClick={ onClick }
-					aria-label={ __( 'Add item', 'elementor' ) }
+					// Translators: %s is the number of kits in the results
+					aria-label={ sprintf( __( 'Add %1$s item', 'elementor' ), labelName?.toLowerCase() ) }
```

Comment:
replace `%1$s` with `%s`. When there is a single placeholder, the best practice is not to use numbers.